frontend/www/js/omegaup/components/contest/Certificates.test.ts

```diff
@@ -0,0 +1,52 @@
+import { mount, shallowMount } from '@vue/test-utils';
+
+import { types } from '../../api_types';
+import T from '../../lang';
+
+import contest_Certificates from './Certificates.vue';
+
+describe('Certificates.vue', () => {
+  it('Should handle generated certificates', () => {
+    const wrapper = shallowMount(contest_Certificates, {
+      propsData: {
+        certificatesDetails: {
+          certificateCutoff: 5,
+          certificatesStatus: 'generated',
+          isCertificateGenerator: true,
+        } as types.ContestCertificatesAdminDetails,
+      },
+    });
```

Comment:
Quizás esta forma de declarar los props aparece en muchos archivos de prueba.Sin embargo, hemos notado que usar `as`, lo que hace es esconder el tipo real.

![image](https://github.com/omegaup/omegaup/assets/3230352/863baa93-c726-41f9-bd3a-f46a4886ed71)

Si te fijas, en la segunda prueba no marca error, aunque se les movió lo mismo.

 La alternativa que puedes usar es la siguiente:
```suggestion
    const certificatesDetails: types.ContestCertificatesAdminDetails = {
      certificateCutoff: 5,
      certificatesStatus: 'generated',
      isCertificateGenerator: true,
    };
    const wrapper = shallowMount(contest_Certificates, {
      propsData: {
        certificatesDetails,
      },
    });
```

También se lo puedes aplicar a la otra prueba